docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -122,6 +122,31 @@ You'll need to replace these values in the policy example below:
 }
 ```
 
+## Step 4. TLS certificates for Teleport
+
+We now need to configure TLS certificates for Teleport to secure its communications.
+
+There are two options when using AWS:
```

Comment:
You can also provide your own certs, right?

Reply:
Yeah - this is mainly just "here are our recommended options for AWS" though. At some point the docs can be updated for using your own certs too, this is just the stuff I've been able to thoroughly test and prove out.

Reply:
Edited this to add "supported options" to indicate that we don't actively support people using their own certs - just `cert-manager` or ACM.